KsApi/graphql-schema.json

```diff
@@ -7290,6 +7302,30 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
```

Comment:
These we're the only changes that occurred when I ran the `bin/apollo-schema-download.sh` script